tempto-core/src/main/java/io/prestosql/tempto/internal/listeners/ProgressLoggingListener.java

```diff
@@ -117,6 +117,9 @@ public void onFinish(ITestContext context)
     private String formatTestName(ITestResult testCase)
     {
         TestMetadata testMetadata = testMetadataReader.readTestMetadata(testCase);
+        if (testMetadata.testGroups.isEmpty()) {
+            return testMetadata.testName;
+        }
```

Comment:
I don't think it's necessary to short-circuit here.
however, it would be better to print `(Groups: [])` when no groups instead of `(Groups: )`.
Maybe we replace `Joiner.on(", ").join` below with just using `testMetadata.testGroups`'s toString?